Makefile

```diff
@@ -1,4 +1,24 @@
+
+IMAGE=segment/mongodb-source
+VERSION=v0.1.3-beta
+
+save-deps:
+	godep save
+
+run:
+	go run main.go mongo.go collection.go config.go description.go $(ARGS)
+
 build:
```

Comment:
@liquidy @tejasmanohar that's because in a separate set of commits we are using vendor instead of godeps workspaces. Maybe upgrading is a bad idea?